eslint/eslint-config/package.json

```diff
@@ -31,16 +31,16 @@
     "@rushstack/eslint-plugin": "workspace:*",
     "@rushstack/eslint-plugin-packlets": "workspace:*",
     "@rushstack/eslint-plugin-security": "workspace:*",
-    "@typescript-eslint/eslint-plugin": "~5.38.0",
-    "@typescript-eslint/experimental-utils": "~5.38.0",
-    "@typescript-eslint/parser": "~5.38.0",
-    "@typescript-eslint/typescript-estree": "~5.38.0",
+    "@typescript-eslint/eslint-plugin": "~5.59.2",
+    "@typescript-eslint/experimental-utils": "~5.59.2",
+    "@typescript-eslint/parser": "~5.59.2",
```

Comment:
This probably breaks the ability to use `@rushstack/eslint-config` with compilers older than a certain version. Make sure the `rush change` bump reflects that appropriately.

Reply:
It actually doesn't. `typescript-eslint` supports all the way back to `3.3.1`.